language_server/src/lsp/lsp_server.cpp

```diff
@@ -309,6 +307,9 @@ void server::on_initialize(const request_id& id, const nlohmann::json& param)
     {
         f->initialize_feature(param);
     }
+
+    if (progress_notification::client_supports_workprogress(param))
```

Comment:
I don't know, I feel like I didn't write any comment lately. So here's at least this 😁

Anyway, this is really minor and maybe it doesn't fit in the VS Code nomenclature

```suggestion
    if (progress_notification::client_supports_work_progress(param))
```